packages/@azure/arm-sqlvirtualmachine/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@azure/arm-sqlvirtualmachine",
+  "author": "Microsoft Corporation",
+  "description": "SqlVirtualMachineManagementClient Library with typescript type definitions for node.js and browser.",
+  "version": "0.1.0",
```

Comment:
Should we keep the version in sync with Node package?

Reply:
Yes, we should. I'll update it.